react/components/EditorContainer/StoreEditor/Styles/StyleEditor/typography/FontEditor.tsx

```diff
@@ -178,13 +204,28 @@ const CustomFont: React.FunctionComponent<Props> = ({
     deleteFont({ variables: { id: id as string } }).then(result => {
       if (
         result == null ||
+        result.data == null ||
         (result.errors != null && result.errors.length > 0)
       ) {
         // TODO: treat erros on delete
         return
       }
-      showToast('Font family removed succesfully.')
-      history.goBack()
+      if (result.data.deleteFontFamily != null) {
+        showToast(
+          intl.formatMessage({
+            id:
+              'admin/pages.editor.styles.edit.font-family.remove-font-success',
+          })
+        )
+        history.goBack()
+      } else {
+        showToast(
```

Comment:
This else resumes the whole fix.
The other changes are just warning removals and other chores.